src/js/components/ToggleButton.js

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React from 'react';
+
+class ToggleButton extends React.Component {
+  render() {
+    let {
+      checkboxClassName,
+      checked,
+      children,
+      className,
+      id,
+      onChange
+    } = this.props;
+
+    return (
+      <span className={classNames(className)}>
+        <input
+          className={classNames(checkboxClassName)}
+          id={id}
+          name="checkbox"
+          onChange={onChange}
+          type="checkbox" />
+        <label
+          checked={checked}
+          htmlFor={id}>
+          {children}
+        </label>
+      </span>
```

Comment:
You could wrap the entire component in a `label` element, then use a `span` in place of what is currently a `label`. This way you don't need to use the `id` and `htmlFor` attributes as the browser will handle that behavior without the explicit `for` attribute. Though, I'm not sure how that might affect accessibility...